packages/nextjs/package.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "@opencreek/eslint-plugin-nextjs",
+    "version": "0.4.0",
+    "description": "ESLint plugin for Common TypeScript rules",
+    "type": "module",
+    "keywords": [
+        "eslint",
+        "eslintplugin",
+        "eslint-plugin"
+    ],
+    "main": "./build/lib/index.js",
+    "files": [
+        "build/**"
+    ],
+    "author": "Opencreek Technology<oss@opencreek.tech>",
+    "repository": "https://github.com/opencreek/eslint-plugin-ts",
```

Comment:
Copy paste

Reply:
This is correct. The repository actually is this (as you can see in the url of this PR :P)